controllers/api/file/linkAction/DeleteFileLinkAction.inc.php

```diff
@@ -27,18 +27,20 @@ class DeleteFileLinkAction extends FileLinkAction {
 	function __construct($request, $submissionFile, $stageId, $localeKey = 'grid.action.delete') {
 		$router = $request->getRouter();
 		import('lib.pkp.classes.linkAction.request.RemoteActionConfirmationModal');
+		$modal = new RemoteActionConfirmationModal(
```

Comment:
Is this just a formatting change? Watch out for space-based indents, and better to exclude this kind of thing from the PR, as it's not related to the SQL Server issue.

Reply:
If I remember correctly, it was more to prevent the UI from freezing on another exception. Don't really have the will (time) to format the code just for the fun of it. ;)

Reply:
Resolved instead by removing the reference from the parent constructor's function definition: https://github.com/pkp/pkp-lib/commit/4807816e5471d6dc399aab5add803c5a08765541